deepchecks/utils/distribution/preprocessing.py

```diff
@@ -159,8 +159,10 @@ def preprocess_2_cat_cols_to_same_bins(dist1: Union[np.ndarray, pd.Series], dist
         list of all categories that the percentages represent.
 
     """
-    categories_list = list(set(dist1).union(set(dist2)))
     dist1_counter, dist2_counter = Counter(dist1), Counter(dist2)
+    len1 = len(dist1)
+    len2 = len(dist2)
```

Comment:
len2 is only used once later on

Reply:
Scratch that, but there is a bug in line 192
ratio should be multiplied by size of dists